### PR TITLE
add readme

### PR DESCRIPTION
**Pull request by DenRomanen**
У меня установлен Prettier для VScod он автоматически  ставит " " вместо ' '.

### CODE REVIEW

**Comment by koras** [comment]
Все сбоки запускаются.
При запросе несуществующего пользователя, приложение падает.

> [nodemon] starting `node app.js`
_http_outgoing.js:455
    throw new ERR_HTTP_HEADERS_SENT('set');
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:455:11)
    at ServerResponse.header (/var/www/www/PCR-3656/Sprint12/node_modules/express/lib/response.js:771:10)
    at ServerResponse.json (/var/www/www/PCR-3656/Sprint12/node_modules/express/lib/response.js:264:10)
    at ServerResponse.send (/var/www/www/PCR-3656/Sprint12/node_modules/express/lib/response.js:158:21)
    at /var/www/www/PCR-3656/Sprint12/routes/users.js:32:21
    at FSReqCallback.readFileAfterClose [as oncomplete] (internal/fs/read_file_context.js:61:3)
[nodemon] app crashed - waiting for file changes before starting...




Не получилось показать всех пользователей

**Comment by DenRomanen** [comment]
Поправил, postman ошибок вроде больше не выдает.

**Comment by koras** [comment]
Ошибка  http://localhost:3000/users/8340d0ec33270a25f2413b69 

https://www.youtube.com/watch?v=7QaMFMV-BeY&feature=youtu.be


`> sprint-12@1.0.0 dev /var/www/www/PCR-3656/Sprint12
> nodemon app.js

[nodemon] 1.19.3
[nodemon] to restart at any time, enter `rs`
[nodemon] watching dir(s): *.*
[nodemon] watching extensions: js,mjs,json
[nodemon] starting `node app.js`
_http_outgoing.js:455
    throw new ERR_HTTP_HEADERS_SENT('set');
    ^

Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client
    at ServerResponse.setHeader (_http_outgoing.js:455:11)
    at ServerResponse.header (/var/www/www/PCR-3656/Sprint12/node_modules/express/lib/response.js:771:10)
    at ServerResponse.json (/var/www/www/PCR-3656/Sprint12/node_modules/express/lib/response.js:264:10)
    at ServerResponse.send (/var/www/www/PCR-3656/Sprint12/node_modules/express/lib/response.js:158:21)
    at /var/www/www/PCR-3656/Sprint12/routes/users.js:33:21
    at FSReqCallback.readFileAfterClose [as oncomplete] (internal/fs/read_file_context.js:61:3)
[nodemon] app crashed - waiting for file changes before starting...
`

File: routes/users.js [hunk 1]
```diff
@@ -27,20 +28,19 @@ const sendUsersId = (req, res) => {
...
-  if (req.params.someRequest) {
-    res.status(404).send({ message: "Запрашиваемый ресурс не найден" });
-    return;
-  }
-});
 
 router.get("/users", sendUsers);
 router.get("/users/:id", sendUsersId);
+router.get("/:someRequest", (req, res) => {
+  res.status(404).send({ message: "Запрашиваемый ресурс не найден" });
```
Comment (koras) [inline routes/users.js]: Ошибки исправлены. Приложение не падает.

**Pull request merged**